messages/german.json

```
@@ -0,0 +1,1385 @@
+{
+    "_metadata": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Metadata",
+        "fallback":  "en",
+        "articles": [
+            {"pattern": "[aeiou]*", "article": "einen"},
+            {"pattern": null, "article": "ein"}
+        ],
+        "plural": [
+            {"number": 1, "index": 0},
+            {"number": null, "index": 1}
+        ],
+        "list": [
+            " und ",
+            ", ",
+            ", und "
+        ]
+    },
+    "_roles": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Roles",
+        "alpha wolf": ["Leitwolf", "Leitwölfe"],
+        "amnesiac": ["Amnesiekranker", "Amnesiekranke"],
+        "assassin": ["Attentäter", "Attentäter"],
+        "augur": ["Weisssager", "Weisssager"],
+        "blessed villager": ["Gesegneter Bürger", "Gesegnete Bürger"],
+        "bodyguard": ["Leibwächter", "Leibwächter"],
+        "clone": ["Klon", "Klone"],
+        "crazed shaman": ["Verrückten Schamanen", "Verrückter Schamane"],
+        "cult leader": ["Kultführer", "Kultführer"],
+        "cultist": ["Kultist", "Kultisten"],
+        "cursed villager": ["Verfluchter", "Verfluchte"],
+        "demoniac": ["Besessener", "Besessene"],
+        "detective": ["Detektiv", "Detektive"],
+        "doctor": ["Arzt", "Ärzte"],
+        "doomsayer": ["Schwarzseher", "Schwarzseher"],
+        "dullahan": ["Kopfloser Ritter", "Kopflose Ritter"],
+        "fallen angel": ["Gefallener Engel", "Gefallene Engel"],
+        "fool": ["Dummkopf", "Dummköpfe"],
+        "guardian angel": ["Schutzengel", "Schutzengel"],
+        "gunner": ["Schütze", "Schützen"],
+        "hag": ["Hexe", "Hexen"],
+        "harlot": ["Dirne", "Dirnen"],
+        "hunter": ["Jäger", "Jäger"],
+        "insomniac": ["Schlafloser", "Schlaflose"],
+        "investigator": ["Inspektor", "Inspector"],
+        "jester": ["Hofnarr", "Hofnarr"],
+        "lycan": ["Lykaner", "Lykaner"],
+        "mad scientist": ["Verrückter Wissenschaftler", "Verrückte Wissenschaftler"],
+        "matchmaker": ["Amor", "Amor"],
+        "mayor": ["Bürgermeister", "Bürgermeister"],
+        "minion": ["Lakai", "Lakaien"],
+        "monster": ["Monster", "Monster"],
+        "mystic": ["Mystiker", "Mystiker"],
+        "oracle": ["Orakel", "Orakel"],
+        "piper": ["Flötenspieler", "Flötenspieler"],
+        "priest": ["Priester", "Priester"],
+        "prophet": ["Prophet", "Propheten"],
+        "seer": ["Seher", "Seherin"],
```

```suggestion
        "seer": ["Seher", "Seher"],
```

Singular und Plural, nicht männlich und weiblich!